### DISCUSSION TITLE
Feature suggestion / help needed with adding signature + date to PDF

### DISCUSSION THREAD

**Discussion by ovizii** [Stirling-Tools/Stirling-PDF#389] [Ideas]
A few times a month, I need to print a PDF, tick a box, add the date and sign it. 
After which I scan it and email it to the recipient. 

I don't need to store these, they are pretty unimportant.

I can't find the right tool in your box. The "sign" function lets me add my scanned signature, but I see no possibility to tick a box or simply place an "X" on top of it. 

Does anyone have any suggestion to solve this situation?

**Comment by Frooodle** [comment]
You could technically have a x as a signature and move it the desired box 

However I see what you mean
There is a missing draw/annotation feature within Stirling-PDF

**Reply by ovizii** [reply]
Thanks for taking a look and now that I had a second look, it looks like I can add as many images as I want to place on top of a PDF. This would take care of the signature and the "X" but what about the date? 

I even went as far as running docuseal to get this done but that is overkill. I only want to simplify my life not overcomplicate it :-) 

So if you have any solution that would be very much appreciated.

**Reply by Frooodle** [reply]
Only annotation support right now is via sign feature 
So for date it would probably be Sign by text and type in the date 

I'll try streamline this more in future

**Reply by ovizii** [reply]
Sorry, the problem was me, not your tool. Unfortunately I assumed I had to choose between: image, draw or text input. 

Now that I see, I can use all 3 multiple times at the same time, my issue is solved. This is all I needed. 

Sorry for being so dense.
